docs/topics/azure-stack.md

```diff
@@ -135,6 +136,110 @@ The `AKS Base Image` marketplace item has to be available in your Azure Stack's
 
 Each AKS Engine release is validated and tied to a specific version of the AKS Base Image. Therefore, you need to take note of the base image version required by the AKS Engine release that you plan to use, and then download exactly that base image version. New builds of the `AKS Base Image` are frequently released to ensure that your disconnected cluster can be upgraded to the latest supported version of each component.
 
+## Azure Monitor for containers
```

Comment:
This section seems to apply to AKSe in general.
Should we update this section https://github.com/Azure/aks-engine/blob/504e1039bc357d949ec4c62dd0fa9375ec182e85/docs/topics/monitoring.md#azure-monitor-for-containers instead? 
Then from the azure stack page we can refer to the monitoring page and highlight the differences

Reply:
Hi, @jadarsie , As we discussed, there are differences between Azure and Azure stack AKS-Engine with regards to Azure Monitor for containers on-boarding and viewing etc.  I think, we can keep them separate and at some point, we can combine them and just cover the differences.

Reply:
This document was initially conceived as a short reference for people targeting AzS. We mainly want to list what's possible, what's not possible and highlight procedural differences between Az and AzS. We add links to the main source of information for people that want to dig deeper on each topic.

I would like people to know that Az Monitor for Containers works on AzS, but this new section is breaking the premise by doubling the page size with mostly not-AzS specific information. I think this new section is out of place. Would you rather create a new `azure-stack-monitoring.md` (or similar) document instead of updating `monitoring.md`?

Reply:
@jadarsie ,  I have updated common monitoring  doc and just referenced that here.